logger.go

```diff
@@ -0,0 +1,14 @@
+package gocql
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+type StdLogger interface {
+	Print(v ...interface{})
+	Printf(format string, v ...interface{})
+	Println(v ...interface{})
+}
+
+var Logger StdLogger = log.New(ioutil.Discard, "", log.LstdFlags)
```

Comment:
Oh you meant the default output. I may be wrong but os.Stderr and os.Stdout are not safe for concurrent use right ?

I've never tried it but I would think if someone uses `log.Println` in a goroutine and in another goroutine a new, different logger writes to os.Stderr there's a chance the output will be mangled.